src/scripting_api/initialization.js

```diff
@@ -68,34 +71,57 @@ function initSandbox({ data, extra, out }) {
     const obj = objs[0];
     obj.send = send;
     obj.doc = _document.wrapped;
+    obj.globalEval = globalEval;
     const field = new Field(obj);
     const wrapped = new Proxy(field, proxyHandler);
     doc._addField(name, wrapped);
     app._objects[obj.id] = { obj: field, wrapped };
   }
 
-  out.global = Object.create(null);
-  out.app = new Proxy(app, proxyHandler);
-  out.color = new Proxy(new Color(), proxyHandler);
-  out.console = new Proxy(new Console({ send }), proxyHandler);
-  out.util = new Proxy(util, proxyHandler);
-  out.border = Border;
-  out.cursor = Cursor;
-  out.display = Display;
-  out.font = Font;
-  out.highlight = Highlight;
-  out.position = Position;
-  out.scaleHow = ScaleHow;
-  out.scaleWhen = ScaleWhen;
-  out.style = Style;
-  out.trans = Trans;
-  out.zoomtype = ZoomType;
+  globalThis.event = null;
+  globalThis.global = Object.create(null);
+  globalThis.app = new Proxy(app, proxyHandler);
+  globalThis.color = new Proxy(new Color(), proxyHandler);
+  globalThis.console = new Proxy(new Console({ send }), proxyHandler);
+  globalThis.doc = _document.wrapped;
+  globalThis.util = new Proxy(util, proxyHandler);
+  globalThis.border = Border;
+  globalThis.cursor = Cursor;
+  globalThis.display = Display;
+  globalThis.font = Font;
+  globalThis.highlight = Highlight;
+  globalThis.position = Position;
+  globalThis.scaleHow = ScaleHow;
+  globalThis.scaleWhen = ScaleWhen;
+  globalThis.style = Style;
+  globalThis.trans = Trans;
+  globalThis.zoomtype = ZoomType;
 
   for (const name of Object.getOwnPropertyNames(AForm.prototype)) {
     if (name !== "constructor" && !name.startsWith("_")) {
-      out[name] = aform[name].bind(aform);
+      globalThis[name] = aform[name].bind(aform);
+    }
+  }
+
+  // The doc properties must live in the global scope too
```

Comment:
I'm guessing it's not possible to make the global scope be the document?

Reply:
For example when I do `console.println(this)` in acrobat, i get `[object Doc]`

Reply:
I think that in a way this code change globalThis into a kind of doc.
About the second point, we can change the implementation of `toString`.
In chrome, it displays `[object global]`.